osi_hostvehicledata.proto

```diff
@@ -2,31 +2,382 @@ syntax = "proto2";
+
+        //
+        // \brief The focus here is on the description of a wheel.
+        //
+        message Wheel
+        {
+            // The axle which contains this wheel.  A value of 0 represents the
+            // front-most axle of the vehicle with higher numbers incrementing
+            // towards the rear-most axle.
+            //
+            optional uint32 axle = 1;
+
+            // The index of the wheel on the axle, counting in the direction
+            // of positive-y, i.e. right-to-left.
+            //
+            // For example, on a standard 2-axle, 4-wheel car, the rear-right
+            // wheel would be (axle=1, index=0).
+            //
+            optional uint32 index = 2;
+
+            // Dry friction is a force that opposes the relative lateral motion of two solid surfaces
+            // in contact. It is subdivided into static friction between non-moving surfaces and kinetic
+            // friction between moving surfaces.
+            // Ued here is the dry friction coefficient of the paired materials (see reference).
+            // Dimensionless.
+            //
+            // Unit: -
+            //
+            // \par References:
+            // - http://adaptivemap.ma.psu.edu/websites/6_friction/dry_friction/dryfriction.html
+            //
+            //optional double kinetic_friction_coefficient = 3;
+
+            // Contact point of the mid of the tire tread with the world.
+            // Absolute coordinates (x, y, z).
+            //
+            // Unit: m
+            //
+            //optional Vector3d contact_point = 4;
+
+            // Contains the rotation rate of the wheel per second in radians.
+            // Based on the processed output of the hall sensor at the wheel.
+            //
+            // Unit: rad/s
+            //
+            optional double rotation_rate = 5;
+
+            // Contains the longitudinal slip of the tire.
+            // \par References:
+            // - https://www.kfz-tech.de/Biblio/Formelsammlung/Schlupf.htm
+            //
+            // Unit: %
+            //
+            optional double slip = 6;
+        }
+    }
+
+    //
+    // \brief Current calculated and estimated location that can be based on GNSS- and related navigation sensors,
+    // but this message does not contain the single sensor values of the sensorics.
+    //
+    // This message contains the most accurate information the vehicle knows about its positioning and kinematics
+    // available on the board net.
+    // Because of this the values can differ from the "true" values calculated out of
+    // GroundTruth::proj_string, GroundTruth::MovingObject::BaseMoving::position, GroundTruth::host_vehicle_id.
+    //
+    message VehiclePositionAndKinematics
+    {
+        // ToDo: Is a boolean necessery which signal is counting?
+ 
+        //
+        // \brief Most accurate, calculated and estimated kinematic data.
+        //
+        message CartesianInformation
```

Comment:
@jruebsam 
Would you use this message, if released?
Otherwise I would leave it out for the beginning.

Reply:
As discussed: origin + relative distance in x,y is used to generate measurements from the localization function of real vehicles.